app/views/criteria/_criterion.html.erb

```diff
@@ -11,8 +11,7 @@
                          length: 27,
                          omission: '...'),
                 edit_assignment_criterion_path(assignment_id: @assignment.id,
-                                               id: criterion.id,
```

Comment:
Removed usages of `criterion_type` from the `edit` and `destroy` requests but kept the ones for `create` and `update` as `criterion_type` is required for the latter two methods.